<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="OpenSerialPort">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SerialPort">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SerialPort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SerialSend">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serialString">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CT + START(Direction, Model, Auto Shutter, Degree, PauseTime, RepeatTimes);&lt;/p&gt;&lt;p&gt;| | |&lt;/p&gt;&lt;p&gt;Direction:  0:Left  1:Right &lt;/p&gt;&lt;p&gt;Model:      0:Continue 1:Intermittent 2:Swing&lt;/p&gt;&lt;p&gt;Auto Shutter:  1:Yes  0:No&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string extracomment="Direction,  Model,  Auto Shutter,  Degree,  PauseTime,  RepeatTimes">CT+START(0,1,0,45,4,1);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Comand</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TOZERO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopRotate">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serialSpeedDown">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SpeedDown</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serialSpeedUp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SpeedUp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serialMute">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sound</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
